contrib/packaging/rpm/skydive.spec

```diff
@@ -30,6 +37,9 @@ Summary:        Real-time network topology and protocols analyzer.
 License:        ASL 2.0
 URL:            https://%{import_path}
 Source0:        https://%{import_path}/releases/download/v%{source}/skydive-%{source}.tar.gz
+Source1:        skydive.pp
```

Comment:
Why putting this file as Source1, Source2, Source3, ..., as they are already part of the original tar.gz